presets/mainnet/electra.yaml

```diff
@@ -38,3 +38,8 @@ MAX_CONSOLIDATIONS: 1
 MAX_DEPOSIT_RECEIPTS_PER_PAYLOAD: 8192
 # 2**4 (= 16) withdrawal requests
 MAX_WITHDRAWAL_REQUESTS_PER_PAYLOAD: 16
+
+# Withdrawals processing
+# ---------------------------------------------------------------
+# 2**3 ( = 8) pending withdrawals
+MAX_PENDING_PARTIALS_PER_WITHDRAWALS_SWEEP: 8
```

Comment:
lots of pluralisation here, we can probably make it easier to read...

`MAX_PENDING_PARTIAL_WITHDRAWALS_PER_SWEEP` would be easier to read, im not set on that specific name but as an example, `PARTIALS` and `WITDRAWALS` together makes the above quite a mouthful!

Reply:
yeah im open to this although I was keeping the `PER_WITHDRAWALS_SWEEP` for symmetry with the other constants we have defined in that code

Reply:
fair, im not sure how the name reads well keeping that bit... maybe we just leave it?

Reply:
going to merge this for now, happy to discuss exact name further in another PR